include/powsybl/iidm/Connectable.hpp

```diff
@@ -22,7 +21,7 @@ namespace iidm {
 
 class Network;
 
-class Connectable : public Identifiable, public MultiVariantObject {
+class Connectable : public Identifiable {
```

Comment:
This class is a merge of Connectable and AbstractConnectable. You should not remove MultiVariantObject from the inheritance tree.